src/api/user/user.seed.js

```diff
@@ -0,0 +1,16 @@
+import User from './user.model';
+
+
+export default function(){
+  // TODO only do this during seeding
+  return User.sync({force: true}).then( () => {
```

Comment:
there's not really a reason to use Promise.then style here, and I agree it would be more consistent to write await

I want to throw out however, that sometimes (not in this case) .then style is more appropriate, e.g. when nested try{ ... } catch () { ... } blocks start to look nasty. The paradigms surrounding await are still developing a bit I think.